stream_alert_cli/version.py

```diff
@@ -106,30 +104,53 @@ def _publish_helper(self, **kwargs):
         client = boto3.client('lambda', region_name=region)
         code_sha_256 = self.config['lambda'][self.package.config_key]['source_current_hash']
 
-        # Publish the function
+        # Publish the function(s)
+        if self.package.package_name == 'stream_alert_app':
```

Comment:
can we keep this class as generic as possible and modify the logic above to work with any `function_name`?

Reply:
for Athena, the clustered vs non clustered logic lives in `deploy.py`

Reply:
I'll try to make this change. I saw that athena handled this in deploy.py but it I couldn't see a reason why forking logic there is better than just forking it here. tbh - the LambdaPackage subclasses should really override methods like this one to handle cases that require different logic.

Reply:
yeah, that's a better solution